src/create_oauth2_client.ts

```diff
@@ -169,6 +169,41 @@ export function createSlackOAuth2Client(
   });
 }
 
+/**
+ * Creates an OAuth 2.0 client with Twitch as the provider.
+ *
+ * Requires environment variables:
+ * 1. TWITCH_CLIENT_ID
+ * 2. TWITCH_CLIENT_SECRET
+ *
+ * ```ts
+ * import { createTwitchOAuth2Client } from "https://deno.land/x/deno_kv_oauth/mod.ts";
+ *
+ * const oauth2Client = createTwitchOAuth2Client({
+ *  redirectUri: "http://localhost:8000/callback",
+ *  defaults: {
+ *    scope: "users.read"
+ *  }
+ * });
+ * ```
+ *
+ * @see {@link https://dev.twitch.tv/docs/authentication/getting-tokens-oauth/#authorization-code-grant-flow}
+ */
+export function createTwitchOAuth2Client(
+  additionalOAuth2ClientConfig?:
```

Comment:
```suggestion
  additionalOAuth2ClientConfig:
```
We should do this for any other functions requiring redirect URI and/or scope in another PR.

Reply:
Yup, makes sense. I'll fix twitch's in this PR and open another one fixing all of them once this gets merged.

Reply:
Cool. I've already done this for the others in #80.

Reply:
Oh nice, sounds good.